integrations/weaviate/src/haystack_integrations/document_stores/weaviate/document_store.py

```diff
@@ -25,6 +25,20 @@
     "weaviate.auth.AuthApiKey": weaviate.auth.AuthApiKey,
 }
 
+# This is the default collection properties for Weaviate.
+# It's a list of properties that will be created on the collection.
+# These are extremely similar to the Document dataclass, but with a few differences:
+# - `id` is renamed to `_original_id` as the `id` field is reserved by Weaviate.
+# - `blob` is split into `blob_data` and `blob_mime_type` as it's more efficient to store them separately.
+DOCUMENT_COLLECTION_PROPERTIES = [
+    {"name": "_original_id", "dataType": ["text"]},
+    {"name": "content", "dataType": ["text"]},
+    {"name": "dataframe", "dataType": ["text"]},
+    {"name": "blob_data", "dataType": ["blob"]},
+    {"name": "blob_mime_type", "dataType": ["text"]},
```

Comment:
what if the blob has a [meta field](https://github.com/deepset-ai/haystack/blob/c47b82c54f313cf269ab0c6d4989e6857fdb082e/haystack/dataclasses/byte_stream.py#L13)?
do we need another field to map it?

Reply:
I omitted it for the same reason I omitted `meta`, though I forgot to properly name the key when converting the Document. I'll handle that.

Reply:
Ah! Nothing to handle in this PR actually, I still have the code locally. 👀